app/src/main/java/org/simple/clinic/search/results/PatientSearchResultsAdapter.kt

```diff
@@ -103,19 +107,25 @@ class PatientSearchResultsAdapter @Inject constructor(
         phoneNumberTextView.text = phoneObfuscator.obfuscate(phoneNumber!!)
       }
 
-      // TODO: Join BP table with patient search query.
-      val lastBpDate: LocalDate? = null
-      val lastBpFacilityName: String? = null
+      val lastBp = searchResult.lastBp
+      if (lastBp == null) {
+        lastBpDateFrame.visibility = View.GONE
 
-      if (lastBpDate == null) {
-        lastBpDateTextView.visibility = View.GONE
```

Comment:
There is an extra line here. Can we remove that?

Reply:
Done